pkg/injector/envoy_container_test.go

```diff
@@ -0,0 +1,77 @@
+package injector
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	tassert "github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/utils/pointer"
+
+	"github.com/openservicemesh/osm/pkg/configurator"
+	"github.com/openservicemesh/osm/pkg/constants"
+)
+
+func Test_getPlatformSpecificSpecComponents(t *testing.T) {
```

Comment:
```suggestion
func TestGetPlatformSpecificSpecComponents(t *testing.T) {
```

Reply:
Good catch! _shakes fist at VSCode's test generation_

Reply:
_sigh_ accepting PR suggestions breaks DCO; of course